### PR TITLE
change the table columns to include ipv4,ipv6

### PR DESCRIPTION
**Pull request by sushmak295**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Review by FumingZhang — COMMENTED** [review]
The change looks good to me, please fix lint issues.

/mnt/vss/_work/1/s/src/aks-preview/azext_aks_preview/_format.py:58:46: W291 trailing whitespace
/mnt/vss/_work/1/s/src/aks-preview/azext_aks_preview/_format.py:59:16: E111 indentation is not a multiple of 4
/mnt/vss/_work/1/s/src/aks-preview/azext_aks_preview/_format.py:60:48: W291 trailing whitespace
/mnt/vss/_work/1/s/src/aks-preview/azext_aks_preview/_format.py:61:16: E111 indentation is not a multiple of 4
/mnt/vss/_work/1/s/src/aks-preview/azext_aks_preview/_format.py:61:47: W291 trailing whitespace
/mnt/vss/_work/1/s/src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py:2183:5: E303 too many blank lines (2)

**Review by FumingZhang — APPROVED** [review]
LGTM

Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=85246796&view=results) to validate the change.

**Comment by yanzhudd** [comment]
Please add changlog to HISTORY.rst. If you want to relase a new version for this change, please update the version in setup.py as well.

File: src/aks-preview/HISTORY.rst [hunk 1]
```diff
@@ -22,6 +22,10 @@ Pending
 * Add `--enable-addon-autoscaling` to the `az aks create` command.
 * Add `--ip-families` to the `az aks update` command.
 
+0.5.175
```
Comment (FumingZhang) [inline src/aks-preview/HISTORY.rst:25]: We've adopted a new naming pattern, please update this to something like `1.0.0b3` and update the `VERSION` in `setup.py` too.
https://github.com/Azure/azure-cli-extensions/pull/7151#discussion_r1446925040

**Comment by zhoxing-ms** [comment]
image
@sushmak295 Please remove the duplicated folder

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**